packages/ember-metal/lib/property_events.js

```diff
@@ -99,15 +99,15 @@ function propertyDidChange(obj, keyName) {
     }
 
     chainsDidChange(obj, keyName, m, false);
-    notifyObservers(obj, keyName);
+    notifyObservers(obj, keyName, m);
   }
 
 
   if (obj[PROPERTY_DID_CHANGE]) {
     obj[PROPERTY_DID_CHANGE](keyName);
   }
 
-  if (obj.isDestroying) { return; }
+  if (m && m.sourceDestroying) { return; }
```

Comment:
Ya we can, but we don't ever set it to `null` manually anymore (used to be nulled out in `deleteMeta`).